.devcontainer/devcontainer.json

```diff
@@ -21,7 +21,12 @@
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"ikuyadeu.r",
```

Comment:
what are these?

Reply:
vscode extensions that turn it into an R ide.